### ISSUE TITLE
*WM: settings invalidated - with missing Sensor

### ISSUE THREAD

**Issue by Tonitom** [vinz486/Fingerprints-reader-R503---R502-esp8266-mqtt-Home-Assistant#7]
The fingerprint sensor is still on the way, so I wanted to prepare the controller already.
But the controller does not appear in the WLAN and I get these messages in the monitor. Is the missing sensor affecting the Wifi Manager?

-----------------------------------------------------------------------------------
Welcome to Fingerprint-MQTT sensor
RESET DEVICE
*WM: settings invalidated
�� THIS MAY CAUSE AP NOT TO START UP PROPERLY. YOU NEED TO COMMENT IT OUT AFTER ERASING THE DATA.

Welcome to Fingerprint-MQTT sensor
Device reset not detected.        
Looking for sensor...KO.    
Sensor not found: check serial connection on green/yellow cables.
��

-----------------------------------------------------------------------------------

And thank you for your project, I've been waiting 5 years for such a product, now I have to build it myself because there's still nothing to buy.

**Comment by vinz486** [comment]
Yes, there is a startup code that need to communicate with the sensor device.

**Comment by Tonitom** [comment]
Thanks.
Good to know that everything is going as expected. Then I'll wait until the sensor is delivered.

Do you know of a good Github project with the same controller together with the RC522 RFID reader? I'm just realising that a whole new world is opening up with MQTT.

**Comment by vinz486** [comment]
Sorry but I don't know that reader.

**Comment by Tonitom** [comment]
Thanks, works great
